apps/starter/app/styles/docs.css

```
@@ -1526,7 +1526,7 @@
 
 .Docs .code .is-highlighted {
   --tw-text-opacity: 1;
-  color: hsl(var(--primary) / var(--tw-text-opacity));
+  color: color-mix(in oklab, var(--primary) calc(var(--tw-text-opacity * 100%), transparent));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)